references/validationFunctions/ajccValidation.js

```diff
@@ -0,0 +1,34 @@
+const validation = () => {
+    function validate() {
+
+        var result = {valid: true, message: "Ok"};
+        
+        /* This is not a required field, so first ensure that it exists */
+        if ($field){
+            /* Contingent on the naming system for tumour staging systems to remain consistent */
+            var stagingName = $name.split('_tumour_staging_system')[0].trim().toLowerCase();
+            /* Perform validation only if an AJCC value was selected */
+            if (/^(AJCC)\b/im.test($field)){
+                var requiredFields = [
+                    `${stagingName}_m_category`,
+                    `${stagingName}_n_category`,
+                    `${stagingName}_t_category`
+                ];
+                var convertedRow = Object.fromEntries(Object.entries($row).map(([fieldName,fieldVal]) => [fieldName.toLowerCase(), fieldVal]));
```

Comment:
for all our variables, we should be using `const` or `let` instead of `var` to declare them. `var` variables end up with a global scope that can cause weird errors, defining things as `const` by default ensures we don't reassign them. If you need to change a variable's value thats when we can use `let` instead.